UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSupplies.json

```diff
@@ -0,0 +1,352 @@
+{
+  "resourceType": "Questionnaire",
+  "id": "UrologicalSupplies",
+  "name": "UrologicalSuppliesOrder",
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/UrologicalSupplies-prepopulation"
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicPatientInfo-prepopulation"
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicPractitionerInfo-prepopulation"
+    }
+  ],
+  "title": "UrologicalSupplies Order Questionnaire",
+  "status": "draft",
+  "date": "2020-06-22",
+  "publisher": "Da Vinci DTR",
+  "item": [
+    {
+      "linkId": "1",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire",
+          "valueCanonical": "questionnaire/patient-info"
+        }
+      ],
+      "type": "display"
+    },
+    {
+      "linkId": "2",
+      "type": "group",
+      "required": false,
+      "text": "Diagnosis",
+      "item": [
+        {
+          "linkId": "2.1",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"UrologicalSuppliesPrepopulation\".RelatedDiagnoese"
```

Comment:
Where do you define this expression? I think the CQL define it as "RelevantDiagnoses" in the prepopulation.